workers/data_refinery_workers/downloaders/sra.py

```diff
@@ -107,20 +114,41 @@ def _download_file_aspera(  download_url: str,
                                          "file from the URL via Aspera: {}").format(download_url)
         return False
 
-
     # If Aspera has given a zero-byte file for some reason, let's back off and retry.
     if os.path.getsize(target_file_path) < 1:
         logger.error("Got zero byte ascp download for target, retrying.",
-                    target_url=download_url,
-                    downloader_job=downloader_job.id)
+                     target_url=download_url,
+                     downloader_job=downloader_job.id)
         time.sleep(300)
-        return _download_file_aspera(   download_url,
-                                        download_job,
-                                        target_file_path,
-                                        attempt + 1
-                                    )
+        return _download_file_aspera(download_url,
+                                     downloader_job,
+                                     target_file_path,
+                                     attempt + 1
+                                     )
+    return True
+
+
+def _are_downloads_ready(original_files: List[OriginalFile]) -> bool:
+    """RNASeq reads can be paired. Makes sure both files are downloaded if applicable.
+    """
+    original_file = original_files[0]
```

Comment:
Can we change the function signature rather than always taking the first of a list